packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -81,6 +83,13 @@ function runConflictFarmTests(opts: IConflictFarmConfig, extraSeed?: number): vo
 					operations: [...opts.operations, obliterateRange],
 				},
 			},
+			{
+				name: "obliterate fields",
+				config: {
+					...opts,
+					operations: [insertField, obliterateField],
```

Comment:
i think we will still want regular inserts and removes here, but they will also need to know about fields, so they don't do partial changes to them. 

I also think the generateOperationMessagesForClients function may need to change to not split fields.

also, what happens if you enable applyOpDuringGeneration

Reply:
With applyOpDuringGeneration enabled, I seem to hit similar bugs that we hit when we initially tried to write these tests (namely, insert failure). There is also usually a case or two where the text on the two clients is the same, but the order of some characters varies between the two.